### ISSUE TITLE
iOS9 occasionally has wrong size

### ISSUE THREAD

**Issue by mattdesl** [stackgl/ray-aabb-intersection#2]
iOS 9.1 has a bug where `window.innerWidth/Height` return the incorrect result (urggghh).

When this occurs, it looks like this:

<img src="https://cloud.githubusercontent.com/assets/1383811/10165504/ad9cd776-668d-11e5-9eee-5091ed7b96a3.PNG" width="50%" />

**Comment by mikolalysenko** [comment]
Whoa, wtf?

**Comment by mattdesl** [comment]
Hmm nevermind.

I was testing locally with `<meta>` while attempting to fix scrolling in #1. 

The bug appears when you have a meta like this. If you refresh the page a few times, the `innerWidth` / `innerHeight` is sometimes twice as large. 

``` html
<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
```

This demo isn't affected since it isn't using `<meta>` ... Unfortunately all my demos are broken because they use `<meta>` and `canvas-fit`. :cry:
